WalletWasabi.Fluent/Views/Wallets/Home/TilesView.axaml

```diff
@@ -0,0 +1,35 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d" d:DesignWidth="800"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.Home.TilesView">
+
+  <ScrollViewer HorizontalScrollBarVisibility="Auto" VerticalScrollBarVisibility="Disabled">
+    <ItemsControl Items="{Binding}" DockPanel.Dock="Bottom">
+      <ItemsControl.Styles>
+        <Style Selector="ContentPresenter">
+          <Setter Property="ContentTemplate">
+            <Setter.Value>
+              <DataTemplate>
+                <Border
+                  CornerRadius="2" BorderThickness="1"
+                  Background="{DynamicResource TileRegionColor}"
+                  BorderBrush="{DynamicResource TileBorderColor}"
+                  Margin="0 0 10 0">
+                  <ContentControl Padding="16" Content="{Binding}" />
```

Comment:
Tiles turned really crowded, adjust the padding to mitigate it.

Master:
![image](https://user-images.githubusercontent.com/16364053/201075180-0d343005-15a2-4e1a-aad6-30f2c9d09c55.png)


Pr:
![image](https://user-images.githubusercontent.com/16364053/201075004-438d5f39-06db-431b-bffb-4a2b723ed1db.png)

Reply:
BTW in the tile views... most of the tiles' controls contain some bottom margin. Those might be removed also